index.js

```diff
@@ -40,13 +41,12 @@ if (functioning) {
   app.quit();
 }
 
-function createMainWindow() {
+function createNewWindow(url, options) {
+  options = options || {};
```

Comment:
Here `options = options || {};` can be removed and include a default value `{}` for the `options` argument at the method definition:

```js
function createNewWindow(url, options = {}) {
```